### DISCUSSION TITLE
Parte 1 formato de pasos

### DISCUSSION THREAD

**Discussion by dela2910** [IIC2133-PUC/2024-2#184] [Tarea 3] [answered]
Hay algún problema si ocupo los pasos en vez de un int, junto todos en una lista con un malloc? 
int *steps = (int *)malloc(x * sizeof(int));

a esto me refiero en vez de 
int s

**Comment by sebacarrascop** [comment, marked as answer]
hola!, puedes modificar como quieras el código base.
Saludos
